src/ert/parsing/config_keywords.py

```diff
@@ -226,45 +263,86 @@ def convert(
                 err = f'Cannot find file or directory "{token.value}" \n'
                 if path != token:
                     err += f"The configured value was {path!r} "
-                raise ConfigValidationError(err, config_file=token.filename)
-            return path
+                raise ConfigValidationError.from_info(
+                    ErrorInfo(
+                        message=err, filename=token.filename, originates_from=token
+                    )
+                )
+            return StringToken(path, token, keyword)
         if val_type == SchemaType.CONFIG_EXECUTABLE:
             path = str(token)
             if not os.path.isabs(token) and not os.path.exists(token):
                 path = shutil.which(token)
-                if path is None:
-                    raise ConfigValidationError(
-                        f"Could not find executable {token.value!r}",
-                        config_file=token.filename,
+
+            if path is None:
+                raise ConfigValidationError.from_info(
+                    ErrorInfo(
+                        message=f"Could not find executable {token.value!r}",
+                        filename=token.filename,
+                        originates_from=token,
                     )
+                )
+
             if not os.access(path, os.X_OK):
                 context = (
                     f"{token.value!r} which was resolved to {path!r}"
                     if token.value != path
                     else f"{token.value!r}"
                 )
-                raise ConfigValidationError(
-                    f"File not executable: {context}",
-                    config_file=token.filename,
+                raise ConfigValidationError.from_info(
+                    ErrorInfo(
+                        message=f"File not executable: {context}",
+                        filename=token.filename,
+                        originates_from=token,
+                    )
                 )
-            return path
-        return str(token)
-
-    def apply_constraints(self, args: List[Any]) -> Union[List[Any], Any]:
-        args = [
-            self.convert(x, i) if isinstance(x, FileContextToken) else x
-            for i, x in enumerate(args)
-        ]
+            return StringToken(path, token, keyword)
+        return StringToken(str(token), token, keyword)
+
+    def apply_constraints(
+        self,
+        args: List[Any],
+        keyword: FileContextToken,
+    ) -> Union[List[Any], Any]:
+        errors = []
+
+        args_with_context = []
+        for i, x in enumerate(args):
+            if isinstance(x, FileContextToken):
+                try:
+                    value_with_context = self.token_to_value_with_context(x, i, keyword)
+                    args_with_context.append(value_with_context)
+                except ConfigValidationError as err:
```

Comment:
We need to add some tests that checks that several errors are collected here without unrelated crashes.